tracing-test-utils/src/main/java/com/palantir/tracing/HtmlFormatter.java

```diff
@@ -85,40 +89,34 @@ public static void render(RenderConfig config) throws IOException {
     }
 
     private void renderChronological(StringBuilder sb) {
-        config.spans().stream()
-                .sorted(Comparator.comparingLong(Span::getStartTimeMicroSeconds))
-                .forEachOrdered(span -> formatSpan(span, false, sb));
+        config.spans().stream().sorted(Comparator.comparingLong(Span::getStartTimeMicroSeconds)).forEachOrdered(
+                span -> formatSpan(span, false, sb));
```

Comment:
We've optimised for the fewest possible number of lines here, but I think this is now less readable.